src/xcode/ENA/TestPlans/AllTests.xctestplan

```diff
@@ -43,6 +43,15 @@
         "language" : "tr",
         "region" : "TR"
       }
+    },
+    {
+      "enabled" : false,
+      "id" : "23BB78C4-766D-4EE2-87E6-E0C288E37D26",
+      "name" : "fake-Klingon",
+      "options" : {
+        "language" : "IDELaunchSchemeLanguageRLO",
+        "region" : "AS"
+      }
```

Comment:
What? 😅

Reply:
Absolutely needed to ensure locale-agnostic tests. All previous and current tests might pass for ze Germanz but not for any other locale 😱! See all the issues above.
If we write future tests and refactor the current failing ones, this might even become the new standard configuration to be used in the CI tests. If THIS thing passes, ANYTHING will.